aci/data_source_aci_vnsabsnode.go

```diff
@@ -97,6 +97,22 @@ func dataSourceAciFunctionNode() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"l4_l7_device_interface_consumer_connector_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"l4_l7_device_interface_provider_connector_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"l4_l7_device_interface_consumer_att_notify": &schema.Schema{
```

Comment:
I would expand the att part (on both)

Reply:
we might also want to order them by the consumers and then the providers (same in doc)